### ISSUE TITLE
couldn't find a valid model for Hoardable::RichText association

### ISSUE THREAD

**Issue by sedubois** [waymondo/hoardable#37]
Thanks for this gem! It's the first versioning gem I find which covers tracking changes to Action Text 👍 

I followed the README [instructions](https://github.com/waymondo/hoardable#action-text) but encounter these errors, what might I be doing wrong? Using Rails 7.0.8 and Hoardable 0.14.2.

```
NameError: Rails couldn't find a valid model for Hoardable::RichText association. Please provide the :class_name option on the association declaration. If :class_name is already provided, make sure it's an ActiveRecord::Base subclass.
from /opt/homebrew/lib/ruby/gems/3.3.0/gems/activerecord-7.0.8/lib/active_record/reflection.rb:441:in `rescue in compute_class'
Caused by NameError: uninitialized constant Page::Hoardable::RichText
from /opt/homebrew/lib/ruby/gems/3.3.0/gems/activerecord-7.0.8/lib/active_record/inheritance.rb:259:in `compute_type'
```

Also the IDE (IntelliJ) complains that `hoardable` isn't recognized.

![Screenshot 2024-01-10 at 15 58 24](https://github.com/waymondo/hoardable/assets/4217871/f2e30c36-91e9-438f-9b11-f8cdbff341c9)

**Comment by waymondo** [comment]
Thanks for the report! I just merged a couple commits related to this.

You are getting that "couldn't find a valid model" because of a chicken or the egg scenario, where we need to wait until `ActionText` is loaded before we can load the `Hoardable` subclass versions, but nothing in your code is autoloading `ActionText` prior to the first hoardable `has_rich_text` call.

Regarding your IDE - I don't know if there's a good cross-LSP-server-compatible way for me to merge in another known option for `has_rich_text`. If that irks you enough, I just introduced `has_hoardable_rich_text` that you could use instead.

I haven't released a new version for the updates in the last month, so for now you can update your Gemfile to:

```
gem "hoardable", "~> 0.15", github: "waymondo/hoardable"
```

Let me know if you still run into issues related to this.

**Comment by waymondo** [comment]
closing as this should be fixed now, and 0.15 has been released to rubygems. feel free to re-open if you are still having trouble with this
